volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -376,13 +329,6 @@ def section_strtab(self):
             return self.strtab
         raise AttributeError("Unable to get strtab")
 
-    @property
```

Comment:
This property needs to be exposed to avoid a bump too....

Reply:
This property was removed from the `module` class, not sure if you meant to write the comment here ?

Reply:
Removing it is a non-additive API change.  That's why I said it needs to still be exposed there for anything that had been using it, otherwise a caller is going to call it expecting it to work and they'll get an ugly `AttributeError`.  The point of the version numbers is to avoid situations like that, so happy for you to bump the version number, or put the property back, but you need to do one of them...

Reply:
Ugh, I see, `module` isn't versioned at all!  5:S  That gets trickier...  5:\

Reply:
It still needs to be exposed then, otherwise any code calling `module_instance.taint_flags_list` is going to throw an error.  This is why I like people to be careful about what they throw into classes like this.  Changing them in the future is tricky...  5:P

Reply:
This property was added in this PR :), it was not present before.

Reply:
If it was only to be used internally, it needed to be called `_taint_flags_list`...

Reply:
That is a good point, however there is no need for bumping as it haven't made it into the framework yet. I'll make sure to make it "private" though.

Reply:
Thanks, sorry, I know it's tricky to get right (and I'm getting lost looking at commit diffs *inside* the PR, so I thought it had already gone in).  Yeah, anything that doesn't need to be accessed from the outside would should be marked with a leading `_`, otherwise it becomes part of the API and you've seen the mess of versioning that comes from that...  5;)